packages/form-js-viewer/test/spec/Form.spec.js

```diff
@@ -117,6 +118,62 @@ describe('createForm', function() {
   });
 
 
+  it('should not submit if disabled', async function() {
+
+    // given
+    const data = {
+      creditor: 'John Doe Company',
+      amount: 456,
+      invoiceNumber: 'C-123',
+      approved: true,
+      approvedBy: 'John Doe'
+    };
+
+    // when
+    const form = await createForm({
+      container,
+      data,
+      schema,
+      properties: {
+        readOnly: true
+      }
+    });
+
+    // when
+    const submission = form.submit();
```

Comment:
Here I'd expect an error to be thrown :arrow_up:.

Reply:
Done (cf. https://github.com/bpmn-io/form-js/pull/85/files#diff-53d9b079c7db646b79736d7ab07cd7d3108049b780d14637f41c188a4f6ffc03R138).